react/FilterNavigator.js

```diff
@@ -159,6 +162,12 @@ const FilterNavigator = ({
     (facet) => facet.selected
   )
 
+  const hasFiltersApplied = window.location.href.split('?').length > 1
```

Comment:
This is not a good way to check the applied filters. It will give you false positives since any query string will trigger this.
I think you can use the `useSearchPage` to get the selectedFilters